src/NLog.WindowsIdentity/NLog.WindowsIdentity.csproj

```diff
@@ -73,6 +73,10 @@ ImpersonatingWrapper Target Docs:
     <Title>NLog.WindowsIdentity for NetStandard 2.0</Title>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
```

Comment:
Should be changed from `net6.0` to `net6.0-windows`

Reply:
Will check it tomorrow but I don't believe it does.

Reply:
![image](https://github.com/NLog/NLog.WindowsIdentity/assets/11509660/41ee8258-cad8-43d5-8a1c-b4a9e40d81e9)

Reply:
you are correct, the platform is also needed so have added it.